eos/form-factors/mesonic.hh

```diff
@@ -107,6 +110,44 @@ namespace eos
             static OptionSpecification option_specification();
     };
 
+    template <>
+    class FormFactors<PToGammaOffShell> :
+        public virtual ParameterUser
+    {
+        public:
+            virtual ~FormFactors();
+
+            // axial current, longitudinal polarization of both currents
```

Comment:
Is this correct? From the notes, I understand that F1 and F2 are admixtures of the longitudinal and transverse polarizations.

It would be useful to refer to eq. (24) of the paper.

Reply:
That is completely correct. From eq. (45) in the paper one can see - as far as I understand - that these are admixtures of the BSZ2015 form factors A12 and A1 which correspond to longitudinal and transverse. Thanks!